components/automate-ui/src/app/modules/desktop/unknown-desktop-duration-counts/unknown-desktop-duration-counts.component.ts

```diff
@@ -71,4 +71,8 @@ export class UnknownDesktopDurationCountsComponent  implements OnInit, OnDestroy
         return duration;
     }
   }
+
+  get hasData(): boolean {
+    return this.countedDurationItems.some(item => item.count > 0);
```

Comment:
Following your earlier precedent, may need this guard--or is it guaranteed to exist?
```suggestion
    return this.countedDurationItems && this.countedDurationItems.some(item => item.count > 0);
```

Reply:
@msorens guaranteed unless the initial reducer state is changed. The other guard only exists because of this `filter` in the subscription chain: 
https://github.com/chef/automate/blob/fb331e709beb9a2bc1901bf43197ea5b0ebd984e/components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts#L132

I considered removing the `filter` in this PR, but since I didn't add it initially I wasn't 100% sure of other possible consequences for its removal.